pkg/test/framework/components/istio/operator.go

```diff
@@ -477,8 +477,14 @@ func deployControlPlane(c *operatorComponent, cfg Config, cluster resource.Clust
 		installSettings = append(installSettings, "--set", "values.global.multiCluster.clusterName="+cluster.Name())
 
 		if networkName := cluster.NetworkName(); networkName != "" {
-			installSettings = append(installSettings, "--set", "values.global.meshID="+meshID,
-				"--set", "values.global.network="+networkName)
+			installSettings = append(installSettings,
+				"--set", "values.global.meshID="+meshID,
+				"--set", "values.global.network="+networkName,
+				// TODO(landow) remove these in favor of a dedicated cross-network gateway deployment
+				// ingress must be enabled for multi-network
+				"--set", "values.gateways.istio-ingressgateway.enabled=true",
+				// prevents gateways from calling gateways and throwing off cross-network traffic
```

Comment:
Will users need to set this?

Reply:
To fix this issue, they need the second flag. Multinetwork will function without it though. I'm going to follow up with a doc/PR for having a separate deployment just for cross-network.